shared/Microsoft.AspNetCore.HttpSys.Sources/RequestProcessing/RequestHeaders.cs

```diff
@@ -131,6 +134,62 @@ public bool TryGetValue(string key, out StringValues value)
             get { return false; }
         }
 
+        long? IHeaderDictionary.ContentLength
```

Comment:
get and set are both wrong. See the HeaderCollection implementation.

Reply:
Can this derive from HeaderCollection?

Reply:
Not without significant work, as RequestHeaders will first access the RequestHeaders.Generated methods then go to the dictionary.